pkg/smokescreen/metrics/metrics_test.go

```diff
@@ -0,0 +1,160 @@
+package metrics
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMetricsTags(t *testing.T) {
+	r := require.New(t)
+
+	t.Run("add custom tags", func(t *testing.T) {
+		metric := "acl.allow"
+		mc := NewNoOpMetricsClient()
+
+		err := mc.AddMetricTags(metric, []string{"globalize"})
```

Comment:
Nonblocking note: it'd be nice to have a better comment on `AddMetricTags` to indicate that (AFAICT) tags added in that way will _always_ be emitted along with the metrics they're attached to.

Reply:
Updated the comment!